tests/modules/token_embedders/embedding_test.py

```diff
@@ -18,15 +20,29 @@ def test_get_embedding_layer_uses_correct_embedding_dim(self):
             embeddings_file.write("word1 1.0 2.3 -1.0\n".encode('utf-8'))
             embeddings_file.write("word2 0.1 0.4 -4.0\n".encode('utf-8'))
         embedding_layer = get_pretrained_embedding_layer(embeddings_filename, vocab)
-        assert embedding_layer.embedding_dim == 3
+        assert embedding_layer.get_output_dim() == 3
 
         with gzip.open(embeddings_filename, 'wb') as embeddings_file:
             embeddings_file.write("word1 1.0 2.3 -1.0 3.1\n".encode('utf-8'))
             embeddings_file.write("word2 0.1 0.4 -4.0 -1.2\n".encode('utf-8'))
         embedding_layer = get_pretrained_embedding_layer(embeddings_filename, vocab)
-        assert embedding_layer.embedding_dim == 4
+        assert embedding_layer.get_output_dim() == 4
 
-    def test_get_embedding_layer_crashes_when_embedding_dim_is_one(self):
+        embedding_layer = get_pretrained_embedding_layer(embeddings_filename, vocab, projection_dim=2)
```

Comment:
Can you add a test with a higher order projected input too - this looks like the most fragile case.